### PR TITLE
Bundling for ventura

### PR DESCRIPTION
**Pull request by sarahayu**

### CODE REVIEW

File: gentle/resample.py [hunk 1]
```diff
@@ -68,7 +68,7 @@ def resample_sox(infile, outfile, offset=None, duration=None):
 def resample(infile, outfile, offset=None, duration=None):
     if not os.path.isfile(infile):
         raise IOError("Not a file: %s" % infile)
-    if shutil.which(FFMPEG):
+    if shutil.which(FFMPEG) or os.path.exists(FFMPEG):
```
Comment (strob) [inline gentle/resample.py:71]: what's the case where `shutil.which(FFMPEG)` returns `None` but the path exists? wouldn't that correspond to a broken symlink?

**Review by strob — COMMENTED** [review]
hey @sarahayu - thanks for bumping the version! i'd love to understand why the FFMPEG path existence change was implemented. perhaps you left that in while debugging?

File: gentle/resample.py (hunk 1 above)
Comment (sarahayu) [reply gentle/resample.py:71]: I found that 'which' doesn't work as expected within a frozen environment. Doing 'which /abs/path/to/ffmpeg' works perfectly fine in a normal file system, but when I tried it after bundling the source it returns empty. I tried googling for explanations but couldn't find anything on this weird behavior. Checking for path existence still works in a frozen environment, however, so I used that as a fallback.

Comment (strob) [reply gentle/resample.py:71]: weird and interesting! a comment in the code might help others understand what's going on, but I'm also fine to merge this in as is

**Pull request merged**